.gitignore

```diff
@@ -6,6 +6,8 @@ dist
 .eggs/
 .tox
 .cache
+env/
```

Comment:
I don’t know the policy of the `shub` project, but I think these ignores that are not mandatory of the project are best suited for the developer’s [global git ignore](http://egorsmirnov.me/2015/05/04/global-gitignore-file.html) or [local git ignore](https://stackoverflow.com/a/1753078/939364).

Reply:
fixed